restructuredtext_lint/cli.py

```diff
@@ -37,24 +40,31 @@ def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
                     'type': error.type,
                     'message': error.message,
                     'full_message': error.full_message,
-                } for error in file_errors)
+                } for error in file_errors if error.level >= level)
 
     if format == 'json':
         stream.write(json.dumps(error_dicts))
 
     if error_occurred:
-        sys.exit(1)
+        sys.exit(2)  # Using 2 for linting failure, 1 for internal error
     else:
-        sys.exit(0)
+        sys.exit(0)  # Success!
 
 
 def main():
     # Set up options and parse arguments
-    parser = argparse.ArgumentParser(description='Lint reStructuredText files')
+    parser = argparse.ArgumentParser(description='Lint reStructuredText files. Returns 0 if all files pass linting, '
+                                     '1 for an internal error, and 2 if linting failed.')
     parser.add_argument('--version', action='version', version=VERSION)
     parser.add_argument('filepaths', metavar='filepath', nargs='+', type=str, help='File to lint')
     parser.add_argument('--format', default='text', type=str, help='Format of the output (e.g. text, json)')
     parser.add_argument('--encoding', type=str, help='Encoding of the input file (e.g. utf-8)')
+    parser.add_argument('--level', default=1, type=int, choices=(1, 2, 3, 4),
```

Comment:
The more I see the `--level 1` as opposed to `--level info`, then more I dislike it. Can we please move to `--level info` in this PR as well? It should be easily done via a `dict`:

```python
#  Load in our dependencies
from docutils.utils import Reporter

# Generate our levels mapping for humans
# http://repo.or.cz/docutils.git/blob/422cede485668203abc01c76ca317578ff634b30:/docutils/docutils/utils/__init__.py#l65
LEVEL_MAP = {
    'debug': Reporter.DEBUG_LEVEL,
    'info': Reporter.INFO_LEVEL,
    'warning': Reporter.WARNING_LEVEL,
    'error': Reporter.ERROR_LEVEL,
    'severe': Reporter.SEVERE_LEVEL,
}

# ...
parser.add_argument('--level', default=LEVEL_MAP['warning'], type=LEVEL_MAP.get, choices=LEVEL_MAP.keys())
```